src/cli/cmds/synth.ts

```diff
@@ -74,5 +130,79 @@ async function fetchValidations(): Promise<ValidationConfig[] | undefined> {
   }
 }
 
+async function createHelmScaffolding(apiVersion: string, chartVersion: string, outdir: string) {
+  const tempHelmStructure = createFolderStructure();
+
+  const substituteValues = {
+    apiVersion: apiVersion,
+    version: chartVersion,
+    app: path.basename(path.resolve()),
+  };
+
+  try {
+    await sscaff(tempHelmStructure, outdir, substituteValues);
+  } catch (error) {
+    throw new Error(`An error occurred during Helm chart creation: ${error}`);
+  }
+
+  if (apiVersion === HelmChartApiVersion.V2 && crdsArePresent(config.imports)) {
+    await addCrdsToHelmChart(outdir);
+  }
+
+  function createFolderStructure(): string {
+    const root = fs.mkdtempSync(path.join(os.tmpdir(), 'helm-scaffolding-'));
+
+    fs.mkdirSync(path.join(root, 'templates'));
+
+    const chartYamlFile = {
+      apiVersion: '{{ apiVersion }}',
+      name: '{{ app }}',
+      version: '{{ version }}',
+      description: 'Generated chart for {{ app }}',
+      type: 'application',
+    };
+
+    fs.outputFileSync(path.join(root, CHART_YAML_FILE), yaml.stringify(chartYamlFile));
+
+    const readmeFile = 'This Helm chart is generated using cdk8s. Any manual changes to the chart would be discarded once cdk8s app is synthesized again with `--format helm`.';
+
+    fs.outputFileSync(path.join(root, README), readmeFile);
+
+    if (config.synth?.chartApiVersion === HelmChartApiVersion.V2 && crdsArePresent(config.imports)) {
+      fs.mkdirSync(path.join(root, 'crds'));
+    }
+
+    return root;
+  }
+}
+
+async function downloadCrds(url: string) {
+  const devUrl = matchCrdsDevUrl(url);
```

Comment:
This deserves a test. Also - can you to reuse the `matchImporter` function instead? Because if we ever support another CRD source, we will now have two places we would have to handle it in.